hyperdb/generate.js

```diff
@@ -4,9 +4,12 @@ const Builder = require('hyperdb/builder')
 
 const SCHEMA_DIR = path.join(__dirname, 'schema')
 const DB_DIR = path.join(__dirname, 'db')
```

Comment:
normally we call the folder "spec", i would not have the generation script inside, that makes it hard to nuke, we normally just have that in the top level

Reply:
see https://github.com/holepunchto/blind-media-mirror/blob/main/build.js